src/main.rs

```diff
@@ -13,10 +13,13 @@ mod events;
 mod manager;
 mod platform;
 mod process;
+mod service;
 mod skylight;
 mod util;
 mod windows;
 
+embed_plist::embed_info_plist!("../assets/Info.plist");
```

Comment:
This is necessary in the (upcoming) distribution phase (via e.g. Homebrew) since a distributed binary has to built in a specific way in order to use AX privileges. Embedding `Info.plist` should be enough to make it so.

See https://crates.io/crates/embed_plist#motivation for what this is doing.